it/uk/gov/hmrc/excisemovementcontrolsystemapi/repository/ExciseNumberQueueWorkItemRepositoryItSpec.scala

```diff
@@ -125,16 +127,17 @@ class ExciseNumberQueueWorkItemRepositoryItSpec extends PlaySpec
         savedWorkItem.item mustBe updatedWI.item
       }
 
+      // The database is storing these as ISODates and so truncated to milliseconds
```

Comment:
just a thought, I wonder if we could use the timestampMillisecond every where here and so remove the other timestamp

Reply:
Done (if I've understood correctly.)